Makefile

```diff
@@ -115,6 +118,7 @@ install_go_sdk::
 install_java_sdk::
 
 install_nodejs_sdk::
+	-yarn unlink --cwd $(WORKING_DIR)/sdk/nodejs/bin
```

Comment:
Is `-yarn` correct?

Reply:
It is, but unsure why leading dash is needed, using command from pulumi-kubernetes `install_nodejs_sdk` target.